### PR TITLE
Update README, aboutUs, and UserGuide

### LINKED ISSUE #8: AboutUs - Anthony Lie
Update AboutUs by Anthony Lie

### LINKED ISSUE #14: README - Update content
Update all contents in README to match our project

### LINKED ISSUE #17: Update User Guide - FAQ
Update User Guide for FAQ.

### PR DESCRIPTION
**Pull request by anthony6401**
Add AboutUs - Anthony Lie. Resolve #8 
Update README content. Resolve #14 
Add UserGuide - FAQ. Resolve #17

### CODE REVIEW

**Review by Ma-Yueran — APPROVED** [review]
Looks good to merge

File: docs/UserGuide.md [hunk 1]
```diff
@@ -159,10 +159,14 @@ _{explain the feature here}_
...
 ## FAQ
-
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous HelloFile home folder.
+
+**Q**: What if the name or the directory of the file I tagged is changed? Can I still access the file using HelloFile?<br>
+**A**: No. HelloFile cannot trace the file if its name or directory is changed, but if you still want to manage the file, you can tag it again.
 
+**Q**: Can tag name be duplicated?<br>
```
Comment (luo-git) [inline docs/UserGuide.md:168]: Maybe you can use an empty line instead of using `<br>`

File: README.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
+##About
+* HelloFile is a desktop app for managing files, optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+Users can tag their file with a nickname to access it easier through the GUI.
+* For the detailed documentation of this project, see the **[HelloFile Product Website](https://ay2021s1-cs2103t-f12-1.github.io/tp/)**.
+
+* This is **a project for CS2103T Software Engineering (SE) class**.<br>
```
Comment (luo-git) [inline README.md]: There is an extra `<br>` at the end, which will increase the gap between this line and the `Acknowledgement`. Is it intended?

**Review by luo-git — CHANGES_REQUESTED** [review]
Some minor changes could be made to standardise markdown format.

File: README.md (hunk 2 above)
Comment (anthony6401) [reply README.md]: Didn't realize that. Thanks!

File: docs/UserGuide.md (hunk 1 above)
Comment (anthony6401) [reply docs/UserGuide.md:168]: I tried using empty line, but the gap between Q and A is too large. I think it is better to use `<br>`.

**Review by luo-git — APPROVED** [review]
Good to go.

**Pull request merged**